### PR TITLE
Using the Authorize attribute with Roles on the WASM Client App

### PR DESCRIPTION
**Pull request by MarianSWA**
fix to work by default

### CODE REVIEW

**Comment by kevinchalet** [comment]
Thanks for your PR. Since the Counter page will no longer be accessible without manually granting the user the "Admin" role, should we update the link in the navigation to something like `Counter (admin-only)`?

File: samples/Balosar/Balosar.Client/Pages/Counter.razor [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿@page "/counter"
+@attribute [Authorize(Roles = "Admin")]
```
Comment (kevinchalet) [inline samples/Balosar/Balosar.Client/Pages/Counter.razor:2]: Should we add a comment indicating that these authorization checks execute client-side - which means they can be bypassed - to encourage folks not to store sensitive stuff in their Razor pages?

Comment (MarianSWA) [reply samples/Balosar/Balosar.Client/Pages/Counter.razor:2]: That's a really good suggestion! I'm pushing the changes in a bit

**Comment by kevinchalet** [comment]
Thanks for your contribution!

**Pull request merged**